docs/build/isc/v1.3/docs/tutorials/defi-lend-borrow-tutorial-part-1.md

```diff
@@ -0,0 +1,266 @@
+# DeFi Lend Borrow : Solidity Contracts
+
+DeFi Lend Borrow is a decentralized finance (DeFi) application that enables users to lend and borrow assets on the Shimmer EVM testnet. The project is built using Solidity and Hardhat, with the core functionality provided by smart contracts.
+
+
+## Prerequisites
+
+- [Node.js](https://nodejs.org) >=  v18.0
+- [Hardhat](https://hardhat.org) >= v2.0.0
+- [npx](https://www.npmjs.com/package/npx)  >= v7.1.0.
+
+## DeFi Lend Borrow contracts architecture Overview
+![architecture diagram](/img/tutorials/defi-lend-borrow/defi-lend-borrow-architecture-diagram.png)
+
+## Set Up
+
+First, create a new directory for the project and navigate into it:
+
+```bash
+mkdir defi-lend-borrow
+cd defi-lend-borrow
+```
+
+Then [bootsrap a new Hardhat project](https://hardhat.org/tutorial/creating-a-new-hardhat-project), by running:
+
+```bash
+npx hardhat init
+```
+
+## Configuration
+
+In the `hardhat.config.js` file, update the `networks` object to include the ShimmerEVM Testnet network configuration, as well as the BNB Testnet network configuration. 
+
+```solidity reference
+https://github.com/iota-community/Defi-lend-borrow/blob/e227092ef1123cf8cff766ee4c786a0c40a62bf3/hardhat.config.ts#L1-L42
+```
+
+## Contracts Overview
+
+### IToken
+
+The `IToken` contract is an ERC20 token that represents an interest-bearing asset with lending and borrowing functionalities.
+
+- **Underlying Asset:** The ERC20 token that this contract wraps.
+- **Interest Rate Model:** Determines the interest rates for borrowing and supplying assets.
+- **ITokenManager:** Manages supported tokens and collateralization.
+
+#### Key Functions
+- Mint: This method allows users to mint `IToken` by depositing a specified amount of the underlying token.
```

Comment:
#### Key Functions

- Mint: This method allows users to mint `IToken` by depositing a specified amount of the underlying token.